torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2642,6 +2642,73 @@ def generator():
     return list(generator())
 
 
+def sample_inputs_nn_pad(op_info, device, dtype, requires_grad, mode, **kwargs):
+    assert mode in ('constant', 'reflect', 'replicate', 'circular')
+    if dtype == torch.bool and mode == 'circular':
+        # test_dtypes fails on ASAN with for the case below
+        # runtime error: load of value 190, which is not a valid value for type 'bool'
+        # Reference: https://github.com/pytorch/pytorch/pull/62814#issuecomment-894156562
```

Comment:
Is this a bug in PyTorch?

Reply:
I think so. Was planning to open an issue to investigate later. Thanks for reminding.